docker/default_compose

```diff
@@ -21,7 +21,7 @@ DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 PATH="$DIR/../bin:$PATH"
```

Comment:
This script is in need of some cleanup but generally isn't actually needed.  It is supposed to merge in docker-compose default values into our etna apps, especially common values such as the edge network settings, but it currently doesn't work great.  Fortunately, it's not really required by our build system.  If I can't think of a good fix to some of its merging issues, I may just trash it.